messaging/src/main/java/org/axonframework/eventhandling/EventHandlerInvoker.java

```diff
@@ -70,5 +73,15 @@ default boolean supportsReset() {
      * Performs any activities that are required to reset the state managed by handlers assigned to this invoker.
      */
     default void performReset() {
+        performReset(NO_RESET_PAYLOAD);
+    }
+
+    /**
+     * Performs any activities that are required to reset the state managed by handlers assigned to this invoker.
+     *
+     * @param resetInfo a {@code R} used to support the reset operation
```

Comment:
resetResourse instead resetInfo?

Reply:
I ended up with `resetContext`. What do you think?